js/mmoda.instrument.js

```diff
@@ -1124,22 +1095,30 @@ function panel_title(srcname, param) {
         // true);
 
         // Collect object name 
-        var allFormData = $("form#mmoda-name-resolve").serializeArray().map(function(item, index) {
+        var allFormData = $("form#mmoda-name-resolve").serializeArray().map(function(item) {
           return (item);
         });
 
         // Collect common parameters
-        var commonFormData = $("form#mmoda-common").serializeArray().map(function(item, index) {
+        var commonFormData = $("form#mmoda-common").serializeArray().map(function(item) {
           if (item.name == 'T1' || item.name == 'T2') {
             item.value = item.value.replace(' ', 'T')
           }
           return (item);
         });
         let active_panel_instrument = $('input[name=instrument]', ".instrument-panel.active").val();
+        var instrument_form_serializeJSON = $($(this)[0]).serializeJSON();
+        //var instrument_form_serializeArray = $($(this)[0]).serializeArray();
+        var instrument_form_serializeArray = [];
+        $.each(instrument_form_serializeJSON, function(param, value) {
+          if (param == 'filters') instrument_form_serializeArray.push({ 'name': param, 'value': JSON.stringify(value) });
```

Comment:
Here, the hardcoded parameter name came back. I did changes in this commit https://github.com/oda-hub/mmoda-frontend-module/pull/228/commits/8e7a1238d667002f947cbeea3c2f7cb8b9243a95